### PR TITLE
Aligned

### PR DESCRIPTION
**Pull request by samyakbvs**
#275

fixed

### CODE REVIEW

**Comment by samyakbvs** [comment]
Sorry for closing the last PR @anshumanv

**Comment by anshumanv** [comment]
Test link?

**Comment by samyakbvs** [comment]
one minute

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/samyakjainbvs-gci17.fossasia.org/

**Comment by samyakbvs** [comment]
Squashing commits please wait

**Comment by samyakbvs** [comment]
Pls merge and also thank you so much @anshumanv and @abishekvashok for helping me in learning something new

**Comment by samyakbvs** [comment]
💯

**Comment by samyakbvs** [comment]
Wait I will be providing a new deployment link

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/samyak-jain-gci17.fossasia.org/

**Comment by anshumanv** [comment]
The image is still not properly scaled, it's shrunk now. Please try to make it into dimensions as other images.

**Comment by samyakbvs** [comment]
Can you please approve like this as at least they are aligned and also only one picture is out of dimension the other one cant be noticed also I have checked everything I can't find anything that restricts their size
Please approve :) @anshumanv

**Comment by samyakbvs** [comment]
To add on, the last image would look overstretched if it is expanded further so take that into consideration too

**Comment by samyakbvs** [comment]
and the commits are finally squashed 👍

**Comment by anshumanv** [comment]
Good going with that, but IMO this PR doesn't really solve the issue.
Please wait for other reviewers to review and see what they have to say.
Thanks

File: css/theme-red.css [hunk 1]
```diff
@@ -3065,7 +3065,7 @@ button[type="submit"] .form-loading {
 }
 /*!---------- 22. IMAGE TILES ----------*/
 img {
-  max-width: 100%;
+  max-width: 100%
```
Comment (ParthS007) [inline css/theme-red.css]: @samyakbvs  please add `;` Issue will be fixed 👍

**Review by ParthS007 — CHANGES_REQUESTED** [review]

**Comment by samyakbvs** [comment]
Done @ParthS007 and thanks

**Comment by samyakbvs** [comment]
pls merge :)

**Comment by ParthS007** [comment]
@anshumanv Have a look, are you facing any Issues now?

**Comment by samyakbvs** [comment]
Do i have to make a new test link or refresh the same @ParthS007

**Review by abishekvashok — CHANGES_REQUESTED** [review]
It seem to be broken in mobile.

**Comment by anshumanv** [comment]
@ParthS007, the image in the last column is still off, should we merge? But then this won't be any substantial contribution(just some minor styling changes) and the task asks for substantial contribution.
image

**Comment by abishekvashok** [comment]
screenshot_2017-12-13-18-15-49-531_com android chrome

**Comment by ParthS007** [comment]
@anshumanv agreed with you , @samyakbvs you have done good job till now but please check responsiveness so that PR can be merged.

File: css/theme-red.css [hunk 2]
```diff
@@ -4259,6 +4259,11 @@ video {
   -webkit-transform: translate3d(0, 0, 0);
   -moz-transform: translate3d(0, 0, 0);
 }
+.masonryFlyIn .masonry-item.fadeIn img{
+    padding: 5px;   max-width: 240px;   max-height: 159px;   min-width: 32px;   min-height: 21px;
```
Comment (ParthS007) [inline css/theme-red.css]: write every property in different line.

**Review by ParthS007 — CHANGES_REQUESTED** [review]

**Comment by samyakbvs** [comment]
@anshumanv  when I opened the link it doesn't show that the last column is off
screen shot 2017-12-13 at 6 36 42 pm

**Comment by samyakbvs** [comment]
screen shot 2017-12-13 at 6 41 16 pm
Also the responsiveness is working fine on the laptop

**Comment by samyakbvs** [comment]
I don't know why it isnt on phone @ParthS007

**Comment by samyakbvs** [comment]
screen shot 2017-12-13 at 7 13 23 pm
Also someone pls tell me how to access the highlighted parts

**Comment by samyakbvs** [comment]
@abishekvashok

**Comment by samyakbvs** [comment]
I have added another commit I hope now my PR is substantial enough

**Comment by samyakbvs** [comment]
@anshumanv

File: index.html [hunk 3]
```diff
@@ -266,7 +266,7 @@ <h6 class="uppercase mb16">Left &amp; Right Variations Included</h6>
 			<p class="lead mb40">
 				We've built three distinct image and text combinations for you to showcase the features of your product or service. All combos come in left and right arrangements.
 			</p>
-			<a class="btn-lg btn" href="#">Explore Foundry</a>
+			<a class="btn-lg btn" href="http://foundry.mediumra.re/chooser.html">Explore Foundry</a>
```
Comment (anshumanv) [inline index.html:269]: What's the need for this?

File: index.html [hunk 4]
```diff
@@ -285,7 +285,7 @@ <h6 class="uppercase mb16">A timeless design to be proud of</h6>
...
```
Comment (anshumanv) [inline index.html:288]: and this?

**Review by anshumanv — CHANGES_REQUESTED** [review]

**Comment by anshumanv** [comment]
@samyakbvs number of commits isn't any criteria for substantial contribution.

File: index.html (hunk 3 above)
Comment (samyakbvs) [reply index.html:269]: I just did because the links were empty

File: index.html (hunk 4 above)
Comment (samyakbvs) [reply index.html:288]: same goes for this one

**Comment by samyakbvs** [comment]
Okay @anshumanv, I will do something else too if you want...

**Comment by samyakbvs** [comment]
@ParthS007 I hope now my PR is good to go?

**Comment by samyakbvs** [comment]
@anshumanv it says explore foundry so I added the link to the foundry

**Comment by anshumanv** [comment]
It's some base content from the template, nowhere related to FOSSASIA.

**Comment by samyakbvs** [comment]
Oh so shall I remove it then?

File: index.html [hunk 5]
```diff
@@ -335,7 +335,7 @@ <h5 class="uppercase mb0">Contemporary Living In The Heart Of Toorak</h5>
 	</div>
 
 	<div class="align-bottom relative-xs text-center">
-		<a class="btn btn-filled btn-lg mb32 mt-xs-40" href="#">Register Interest</a>
+		<a class="btn btn-filled btn-lg mb32 mt-xs-40" href="https://www.realestate.com.au/property/53-hopetoun-rd-toorak-vic-3142">Register Interest<a>
```
Comment (abishekvashok) [inline index.html:338]: Why should you include this link?

**Comment by samyakbvs** [comment]
Please don't close the PR

**Comment by samyakbvs** [comment]
Please now see
@anshumanv @ParthS007

**Comment by anshumanv** [comment]
Images are still messed up @samyakbvs 
whatsapp image 2017-12-13 at 11 55 59 pm

**Comment by nzec** [comment]
I don't think this is even required. The current website looks good already.

**Comment by samyakbvs** [comment]
How should I deal with it @anshumanv

**Comment by anshumanv** [comment]
Experiment with styling and see what works, I think there's not much problem with the current setup.
I encourage you to try however you can close this if unable.

**Comment by ParthS007** [comment]
@samyakbvs I agree with @anshumanv , Please close this PR and you can find other bugs and made PR solving them.Thanks 👍

**Pull request closed without merging**